app/src/main/java/cto/its/ny/gov/fakenews/MainActivity.kt

```diff
@@ -1,59 +1,51 @@
 package cto.its.ny.gov.fakenews
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import android.widget.TextView
-import androidx.core.view.allViews
-import androidx.core.view.children
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 import cto.its.ny.gov.fakenews.databinding.ActivityMainBinding
-import cto.its.ny.gov.fakenews.databinding.ArticleViewBinding
 import cto.its.ny.gov.fakenews.model.Article
-import cto.its.ny.gov.fakenews.util.services.NewsService
-import cto.its.ny.gov.fakenews.views.ArticleView
+import cto.its.ny.gov.fakenews.views.adapter.ArticleViewListAdapter
 
-class MainActivity : AppCompatActivity() {
+class MainActivity : AppCompatActivity() , ArticleViewListAdapter.ArticleClickListener{
     private lateinit var binding: ActivityMainBinding
-    private var arrayOfArticles = arrayListOf<Article>()
+    lateinit var itemListRecyclerView: RecyclerView
+
+    companion object{
+       const val INTENT_DETAILS_KEY = "details"
+    }
```

Comment:
Nice work here having the MainActivity implement the ArticleClickListener 🎉 

super minor, but `companion object`s are usually recommended to go at the [bottom of the class](https://kotlinlang.org/docs/coding-conventions.html#class-layout).